File: Baekjoon/1316.py
import sys
num = int(sys.stdin.readline())

cnt = 0
for i in range(num):
    str = sys.stdin.readline().rstrip()
    
    check = 1
    c = str[0]
    for j in range(len(str)):
        if(c != str[j]):
            for k in range(j - 1):
                if(str[k] == str[j]):
                    check = 0
                    break
        c = str[j]
        
    if check == 1:
        cnt = cnt + 1

print(cnt)

File: Baekjoon/1316.cpp
#include <iostream>
#include <string>
using namespace std;

int main(){
    int num;
    cin >> num;

    int cnt = 0;
    string str;
    for(int i = 0; i < num; i++){
        cin >> str;
        int len = str.length();
        
        int check = 1;
        char c = str[0];
        for(int j = 1; j < len; j++){
            if(c != str[j]){
                for(int k = j - 2; k >= 0; k--){
                    if(str[k] == str[j]){
                        check = 0;
                        break;
                    }
                }
            }
            c = str[j];
        }
        if(check == 1){
            cnt += 1;
        }
    }
    cout << cnt;
}
